src/Providers/Amazon.Sagemaker/src/SageMakerSettings.cs

```diff
@@ -0,0 +1,82 @@
+// ReSharper disable once CheckNamespace
+
+namespace LangChain.Providers.Amazon.SageMaker;
+
+public class SageMakerSettings : ChatSettings
+{
+    public new static SageMakerSettings Default { get; } = new()
+    {
+        User = ChatSettings.Default.User,
+        ContentType = "application/json",
```

Comment:
Not sure about this, can it really be changed?

Reply:
you mean if someone passes in a different contentType?  yes, it can be changed.  try it from the test